apps/judicial-system/backend/src/app/modules/case/case.service.ts

```diff
@@ -715,6 +716,14 @@ export class CaseService {
     theCase: Case,
     user: TUser,
   ): Promise<void> {
+    // If case was appealed in court we don't need to send these messages
+    if (
+      theCase.accusedAppealDecision === CaseAppealDecision.APPEAL ||
+      theCase.prosecutorAppealDecision === CaseAppealDecision.ACCEPT
+    ) {
+      return Promise.resolve()
```

Comment:
If you make the method `addMessagesForAppealedCaseToQueue` async you don't need to explicitely return a promise. You can do a simple return instead.

Reply:
However, this check is done a bit prematurely. We still need to post all the case file delivery messages to queue. The only message that should not be queued is the notification message.